utils/eth1.go

```diff
@@ -256,9 +256,9 @@ func FormatAddressLong(address string) template.HTML {
 	}
 	address = FixAddressCasing(address)
 	test := `
```

Comment:
why is this named `test`?

Reply:
To **test** you code review skills.

But seriously, I don't know, I did not choose the name. But of course, your implied "Please use a meaningful name for this" is still valid so I've changed the name accordingly.

I even fixed it at in a neighboring function. AND I have made both functions more performant for some cases.

All thanks to you :heart: